specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/2020-01-15-generalAvailability/CommunicationIdentity.json

```diff
@@ -0,0 +1,264 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationIdentityClient",
+    "description": "Azure Communication Identity Service",
+    "contact": {
+      "email": "acsdevexdisc@microsoft.com"
+    },
+    "version": "2020-01-15-generalAvailability"
```

Comment:
let's change the version to `2020-01-15-preview3`

Reply:
Updated